packages/Panel/stories/examples/Showcase.js

```diff
@@ -27,19 +27,25 @@ const sidePanelProps = () => ({
   zIndex: number("zIndex", undefined, {}, sidePanelGroup),
   offset: object("offset", { top: 0, left: 0, right: 0 }, sidePanelGroup),
   pushContent: boolean("pushMain", false, sidePanelGroup),
+  size: select(
+    "size",
+    [Panel.types.size.MEDIUM, Panel.types.size.LARGE],
+    Panel.types.size.MEDIUM,
+    sidePanelGroup
+  ),
+  
```

Comment:
we no longer need the `isCompact: boolean("isCompact", false, sidePanelGroup),` on line 17